.github/workflows/ci.yml

```diff
@@ -21,6 +21,8 @@ jobs:
         uses: joschi/setup-jdk@v1.0.0
         with:
           java-version: openjdk${{ matrix.java }}
+      - name: Build Quarkus master
+        run: git clone https://github.com/quarkusio/quarkus.git && cd quarkus && mvn -B clean install -DskipTests -DskipITs
```

Comment:
Do we need to `cd ..` for the next action, or does GitHub do that automatically?

Reply:
It starts from the root